### PR TITLE
Added note to clarify globbing support

### PR DESCRIPTION
**Pull request by Nyami**
## Summary

#6293 confirms this behaviour so thought it valuable to add a clear note.

### CODE REVIEW

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks, @Nyami, for adding the note. I've proposed some changes for you to consider.

File: docs/core/tools/dotnet-sln.md [hunk 1]
```diff
@@ -89,3 +89,6 @@ Add multiple C# projects to a solution using a globbing pattern:
 Remove multiple C# projects from a solution using a globbing pattern:
 
 `dotnet sln todo.sln remove **/*.csproj`
+
+> [!NOTE]
+> Globbing is not a CLI feature but rather a feature of the command shell, to sucessfully expand the files you must be using a shell that supports.
```
[the remark this answers is not in the document]
Comment (pkulikov) [reply docs/core/tools/dotnet-sln.md]: one more suggestion: 
- "you must be using" --> "use" or "you must use"

**Review by Thraka — COMMENTED** [review]
Would it be helpful to add a `For more information about globbing, see [Wikipedia](https://en.wikipedia.org/wiki/Glob_%28programming%29).` at the end of the note?

**Comment by rpetrusha** [comment]
This looks good, @Nyami. Though I like the suggestion from @Thraka that we add a link to more information about globbing. Would you like to do that?

**Comment by Nyami** [comment]
@rpetrusha I'm happy to add @Thraka suggestion to the note.

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
Thanks, @Nyami, for the contribution and the additional changes. I'll merge your PR now. Your changes should be live on docs.microsoft.com in the next day or two.

**Pull request merged**